### PR TITLE
Finish mobile first adding projects sections

### PR DESCRIPTION
**Pull request by enocol**
- added and design the Works section.
- Added and design the about myself section.
- Included button interactions.
- used Grid css for the outer layout 
- Used flexBox for inner elements layout

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @enocol , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

- [ ] Your readme has the live demo section without a deployed version. Kindly delete this section / deploy the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
         <li><i class="fa-brands fa-facebook-f"></i></li>
       </ul>
     </section>
+    <section class="card">
```
Comment (souad988) [inline index.html:61]: Great work was done so far 👍🏼  , I want just to mention that the background color you used for this section is diffrent from the template kindly fixe it 

image (34)

image (33)

File: index.html [hunk 2]
```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
...
+              <ul class="listA">
+                <li class="listA-items">BackEnd</li>
+                <li class="listA-items">2015</li>
+              </ul>
+            </div>
+
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
```
Comment (souad988) [inline index.html:80]: This text looks a bit bolder than it should be please do your best to make your design looks exactly like the template

File: index.html [hunk 3]
```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
...
+            <ul class="listB">
+              <li class="listB-items">HTML</li>
+              <li class="listB-items">Css</li>
+              <li class="listB-items">JavaScript</li>
+            </ul>
+            <button>See Project</button>
```
Comment (souad988) [inline index.html:86]: Good job your buttons looks exactly like the template , only make sure to add hover and press effects as requested 

image (35)

File: index.html [hunk 4]
```diff
@@ -58,5 +58,179 @@ <h1>I'm Collins. Glad to see you!</h1>
...
+          </p>
+        </div>
+        <span class="socialmedia">LETS CONNECT</span>
+        <ul class="media">
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin-in"></i></li>
+          <li><i class="fa-brands fa-github"></i></li>
+          <li><i class="fa-brands fa-facebook-f"></i></li>
+        </ul>
+        <button class="resume-button">Get my resume</button>
```
Comment (souad988) [inline index.html]: the about myself section looks good 👍🏼  , some small things to fixe to make it perfect, the vertical spacing between these two buttons and the social links list is not equal, the text in the paragraph and in Lets connect  is a bit bolder than the design and the spacing between the social icons should be increased.

image (36)
image (37)

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @enocol , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868


## Highlights
Good PR description
Use grid and flexbox to layout components
No linter issues


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — APPROVED** [review]
Hi @enocol , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**